internal/collector/sysinfo/platform/platform_linux.go

```diff
@@ -115,11 +118,20 @@ func (p Collector) interopEnabled() (enabled bool) {
 
 // collectWSL collects information about Windows Subsystem for Linux.
 func (p Collector) collectWSL() WSL {
-	info := WSL{Arch: p.getWSLVersion()}
-	if info.Arch == 0 {
+	info := WSL{SubsystemVersion: p.getWSLVersion()}
```

Comment:
Future selves will likely wonder whether `getWSLVersion()` returns the subsystem version (1 or 2) or the WSL application version. I think it's worth renaming it too.